pylib/gyp/generator/ninja.py

```diff
@@ -654,18 +654,18 @@ def WriteActions(
             description = self.GenerateDescription(
                 "ACTION", action.get("message", None), name
             )
-            is_cygwin = (
-                self.msvs_settings.IsRuleRunUnderCygwin(action)
-                if self.flavor == "win"
-                else False
+            win_shell_flags = (
+                self.msvs_settings.GetRuleShellFlags(action)
+                if self.flavor == 'win'
```

Comment:
```suggestion
                if self.flavor == "win"
```
Our Python code is formatted with psf/black which prefers double quotes.